src/codeu/chat/server/Server.java

```diff
@@ -42,245 +42,251 @@
 
 public final class Server {
 
-  private interface Command {
-    void onMessage(InputStream in, OutputStream out) throws IOException;
-  }
+	private interface Command {
+		void onMessage(InputStream in, OutputStream out) throws IOException;
+	}
+
+	private static final Logger.Log LOG = Logger.newLog(Server.class);
 
-  private static final Logger.Log LOG = Logger.newLog(Server.class);
+	private static final int RELAY_REFRESH_MS = 5000; // 5 seconds
```

Comment:
I tried followed the directions on the README for using google-java-format. It seems like it worked but when I try to commit my changes it says that there is nothing to commit. Am I missing a step? Aside from that, I have been using eclipse to edit the code and I tried to change the style preferences but it still shows up wrong on github. Would you recommend me to just stick with vim or something else?